src/main/java/com/somemore/recruitboard/dto/command/RecruitBoardCreateCommandRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package com.somemore.recruitboard.dto.command;
```

Comment:
```suggestion
package com.somemore.recruitboard.dto.request;
```

저희 컨벤션이 위와 같지 않나요?
비슷한 이유로 레코드명에 Command가 들어가는 것에 대해서도 여쭤보고 싶습니다.

Reply:
음.. 컨벤션상으로 request가 맞긴합니다.
다만 해당 dto는 `RecruitBoardUseCase`에서 `recruitBoardCommandService`의 create() 메서드의 인자로 들어가는 dto 이기 때문에 명명을 다르게 하는게 맞다고 생각해서 작성했습니다.

제 나름대로의 규칙을 정한건.. dto.request 패키지는 컨트롤러에서 사용되는 dto, dto.command 패키지는 파사드 서비스에서 다른 서비스를 호출할 때 dto 정보가 달라질 경우 필요 정보만 담는 dto가 모인 패키지로 생각하고 넣긴했는데, 이부분 의논해보면 좋을 것 같습니다.

레코드명에 Command가 들어가는 이유도 비슷한 dto 패키지를 나눈 맥락과 동일하다고 말씀드릴수 있을 것 같아요.